### PR TITLE
Updated to Akka 1.0.7

### PR DESCRIPTION
**Pull request by alexvaluyskiy**
- updated to Akka.Persistence 2.4.2,
- added query tests
- changed type on created_at column from Timestamp to BIGINT
- removed created_at_ticks column

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,80 +1,122 @@
-## Akka.Persistence.PostgreSql
+## Akka.Persistence.PostgresSql
```
Comment (Horusiath) [inline README.md]: It's PostgreSql ;)

File: src/Akka.Persistence.PostgreSql.Tests/PostgreSqlSnapshotStoreSpec.cs [hunk 2]
```diff
@@ -46,5 +50,20 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
             DbUtils.Clean();
         }
+
+        [Fact]
+        public void SnapshotStore_should_save_and_overwrite_snapshot_with_same_sequence_number()
```
Comment (Horusiath) [inline src/Akka.Persistence.PostgreSql.Tests/PostgreSqlSnapshotStoreSpec.cs]: Shouldn't this be a part of `SnapshotStoreSpec` base class?

Comment (alexvaluyskiy) [inline src/Akka.Persistence.PostgreSql.Tests/PostgreSqlSnapshotStoreSpec.cs]: It should be, but this test is skipped in Sql.Common. I have written suggestions about Sql.Common here
https://github.com/akkadotnet/Akka.Persistence.SqlServer/pull/24

**Comment by alexvaluyskiy** [comment]
When I update to Npgsql 3.0.5, these tests start failing. I need help to find out why.
```
Journal_queried_on_PersistenceIdRange_returns_events_for_particular_persistent_ids
Journal_queried_on_Timestamp_returns_events_occurred_before_To_value
Journal_queried_using_multiple_hints_should_apply_all_of_them
Journal_queried_on_Timestamp_returns_events_occurred_after_or_equal_From_value
```

**Comment by Danthar** [comment]
Seems like there is a build script error here. I have restarted the build to be sure.

**Comment by alexvaluyskiy** [comment]
@Danthar nothing changed

**Comment by Danthar** [comment]
I fixed the build system. This pr now needs a rebase. Please wait for @Horusiath to fix his PR. ill merge his first. That way you only have to rebase once.

**Comment by Danthar** [comment]
Ok scratch that. The build fix does not need a rebase. But you got 6 failing unit tests now.

**Pull request merged**